vir/defs/high/ast/expression.rs

```diff
@@ -259,3 +261,12 @@ pub struct Downcast {
     pub field: FieldDecl,
     pub position: Position,
 }
+
+impl Expression {
```

Comment:
This should be in [this file](https://github.com/viperproject/prusti-dev/blob/master/vir/defs/high/operations_internal/expression.rs).